packages/libs/eda/src/lib/map/analysis/appState.ts

```diff
@@ -62,10 +62,11 @@ export const AppState = t.intersection([
 // eslint-disable-next-line @typescript-eslint/no-redeclare
 export type AppState = t.TypeOf<typeof AppState>;
 
-const defaultAppState: AppState = {
+// export default viewport for custom zoom control
+export const defaultAppState: AppState = {
   viewport: {
     center: [0, 0],
-    zoom: 4,
+    zoom: 1,
```

Comment:
Cool - let's keep this and then investigate zoom zero separately on the other ticket (much less urgent)